### PR TITLE
Refresh README

### PR DESCRIPTION
**Pull request by falzm**
This brings a long overdue refresh to the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,24 +13,16 @@ log.SetOutput(r)
...
- - ``metrics``: `Metrics`_,
- - ``sentry``: `Crash reporting`_.
- - ``prefix``: `Prefix`_.
-
 ### Logging
 
 Here is an example configuration:
 
-```yaml
+``yaml
```
Comment (mping-exo) [inline README.md]: Aren't you missing a  "`" sign on opening and closing?

File: README.md [hunk 2]
```diff
@@ -149,9 +168,9 @@ API to acknowledge the event, set `wait: true`.
 
 ### Prefix
 
-The ``prefix`` of the logs and metrics.
+The `prefix` of the logs and metrics.
 
-```yaml
+``yaml
```
Comment (mping-exo) [inline README.md]: Aren't you missing a "`" sign on opening and closing?

File: README.md (hunk 1 above)
Comment (falzm) [reply README.md]: Good catch, missed this one. Fixed!

File: README.md (hunk 2 above)
Comment (falzm) [reply README.md]: Good catch, missed this one. Fixed!

**Pull request merged**